lib/steep/services/signature_help_provider.rb

```diff
@@ -11,15 +11,10 @@ def env
         subtyping.factory.env
       end
 
-      def initialize(source:, subtyping:)
+      def initialize(source:, content:, subtyping:)
```

Comment:
I don't know adding a new argument here is a better way. It seems also better to add a `content` attribute to `Steep::Source` object. Please let me know your opinion.